opencadd/structure/superposition/engines/mmligner.py

```diff
@@ -57,6 +58,21 @@ def __init__(self, executable=None, protein_selector="protein"):
             "Current MMLigner wrappers produces accurate RMSD values but slightly shifted structures!"
         )
 
+    def _safety_checks(self):
+        """
+        Check if `mmligner` is installed (executable found?).
+
+        Raises
+        ------
+        OSError
+            Raises error if executable `mmligner` cannot be found.
+        """
+
+        theseus = find_executable("mmligner")
+        if theseus is None:
+            raise OSError("theseus cannot be located. Is it installed?")
```

Comment:
should be the following:
```suggestion
        mmligner = find_executable("mmligner")
        if mmligner is None:
            raise OSError("mmligner cannot be located. Is it installed?")
```